e2e/wallet_helper.go

```diff
@@ -54,28 +52,15 @@ type CliWallet struct {
 	txOpts           *bind.TransactOpts
 	Owner            common.Address
 	backend          *ethclient.Client
-	identityRegistry abigen.IdentityPromisesTransactorSession
-	tokens           mysttoken.MystTokenTransactorSession
+	identityRegistry bindings.RegistryTransactorSession
```

Comment:
`identityRegistry` is unused. 
func `registerIdentity` at 199 line is unused too.